korman/ui/ui_toolbox.py

```diff
@@ -44,6 +44,12 @@ def draw(self, context):
         col.label("Plasma Pages:")
         col.operator("object.plasma_move_selection_to_page", icon="BOOKMARKS", text="Move to Page")
         col.operator("object.plasma_select_page_objects", icon="RESTRICT_SELECT_OFF", text="Select Objects")
+        
+        col.label("Sound Export:")
+        col.operator("object.plasma_toggle_sound_export", icon="MUTE_IPO_OFF", text="Enable All").enable = True
+#        all_sounds_export = all((i.data.sound.plasma_sound.package for i in bpy.context.selected_objects if i.plasma_modifiers.soundemit))
```

Comment:
Here's a doozy of a one liner:

```suggestion
        all_sounds_export = all((i.package for i in itertools.chain.from_iterable(i.plasma_modifiers.soundment.sounds for i in bpy.context.selected_objects if i.plasma_modifiers.soundemit.enabled)))
```

You'll also need to add `import itertools` to the top of the file.